src/components/screens/home/home-screen.tsx

```diff
@@ -1,7 +1,13 @@
+/**
+ * @module HomeScreen
+ * @date 9/15/2023 - 11:39:05 AM
+ *
+ * @returns {*}
+ */
 const HomeScreen = () => {
```

Comment:
I don't think this component is actually being rendered, but perhaps this does not need to be resolved as part of this feature branch, but feel free to resolve if its obvious to you why it's not being rendered

Reply:
Not rendered no, but it's hooked to the App router right now. Should I remove and let the next feature handle it?